src/store/config.ts

```diff
@@ -0,0 +1,59 @@
+import fs from 'fs';
+
+import {AppContext} from 'mattermost-redux/types/apps';
+
+import {jsonConfigFileStore} from '../utils';
+
+type AppStore = {
+    bot_access_token: string;
+    oauth2_client_secret: string;
+    mm_site_url: string;
+}
+
+interface Store {
+    getBotAccessToken(): string;
+    getSiteURL(): string;
+}
+
+class JSONFileStore implements Store {
+    storeData: AppStore;
+
+    constructor() {
+        this.storeData = {
+            bot_access_token: '',
+            oauth2_client_secret: '',
+        };
+
+        if (fs.existsSync(jsonConfigFileStore)) {
+            fs.readFile(jsonConfigFileStore, (err, data) => {
+                if (err) {
+                    throw err;
+                }
+                this.storeData = JSON.parse(data.toString());
+            });
+        }
```

Comment:
I think we may need to use `readFileSync` here, or have an `async init()` function that reads the json data from the file, so the caller can call that with `await` and have access to the store's data right after the `Store` is instantiated.

Reply:
https://github.com/mattermost/mattermost-app-zendesk/issues/15